include/inout.h

```diff
@@ -140,4 +140,7 @@ constexpr io_port_t port_num_virtualbox = 0x5654u;
 constexpr io_port_t port_num_vmware    = 0x5658u;
 constexpr io_port_t port_num_vmware_hb = 0x5659u; // high bandwidth
 
+// Adlib audio card
+constexpr io_port_t port_num_adlib_command = 0x388u;
```

Comment:
I actually made a TODO comment about using a nicer C++ style for these constants, so you might as well start doing that now 😄 

So we'd use `Port::AdLib::Command` instead.

```ini
namespace Port {

namespace AdLib {

constexpr io_port_t Command = 0x388u;

}
} // namespace Port
```

Reply:
OK! I'll use that - thanks